test/auto_load_tests/fixtures.exs

```diff
@@ -3,6 +3,11 @@ defmodule AutoLoadFixtures do
   use ExUnitFixtures.AutoImport
 
   deffixture not_top_level_fixture do
+
+    on_exit fn ->
+      1
```

Comment:
Actually, not sure how to write a test to do this. From looking at the code it looks pretty clear that it should work though...